### ISSUE TITLE
Low-level error when attempting to decompress  LZW encoded PDF

### ISSUE THREAD

**Issue by dwarring** [pdf-raku/PDF-raku#30]
I've found a particular PDF with LZW encoding that dies when attempting to decompress (see attached);

```
$ raku -I . bin/pdf-rewriter.raku --uncompress /tmp/000094.pdf 
opening /tmp/000094.pdf ...
uncompressing ...
Cannot unbox a type object (Any) to int.
  in method decode at /home/david/git/PDF-raku/lib/PDF/IO/Filter/LZW.rakumod (PDF::IO::Filter::LZW) line 64
  in method decode at /home/david/git/PDF-raku/lib/PDF/IO/Filter/LZW.rakumod (PDF::IO::Filter::LZW) line 24
  in method decode-item at /home/david/git/PDF-raku/lib/PDF/IO/Filter.rakumod (PDF::IO::Filter) line 24
  in method decode at /home/david/git/PDF-raku/lib/PDF/IO/Filter.rakumod (PDF::IO::Filter) line 13
  in method decode at /home/david/git/PDF-raku/lib/PDF/COS/Stream.rakumod (PDF::COS::Stream) line 83
  in block  at /home/david/git/PDF-raku/lib/PDF/COS/Stream.rakumod (PDF::COS::Stream) line 60
  in method uncompress at /home/david/git/PDF-raku/lib/PDF/COS/Stream.rakumod (PDF::COS::Stream) line 106
  in block  at /home/david/git/PDF-raku/lib/PDF/IO/Reader.rakumod (PDF::IO::Reader) line 873
  in method recompress at /home/david/git/PDF-raku/lib/PDF/IO/Reader.rakumod (PDF::IO::Reader) line 857
  in sub MAIN at bin/pdf-rewriter.raku line 62
  in block <unit> at bin/pdf-rewriter.raku line 5
```

Other LZW PDF compressed files seem to work ok. 
[000094.pdf](https://github.com/pdf-raku/PDF-raku/files/6837264/000094.pdf)

Either this is valid, and there's something wrong with LZW decompression, or it's not and this is a LTA error.

More investigation needed.

**Comment by dwarring** [comment]
Fixed on master.

**Comment by dwarring** [comment]
Fix released with PDF 0.4.15
